### PR TITLE
Data preservation

### PR DESCRIPTION
**Pull request by denigit5**
- [ ] Added a data preservation code on my project.
- [ ] This pull request has one contributor since my partner has not shown up as we had agreed and did not communicate sooner.
- [ ] I have  reached out to student success to resolve the issue.
- [ ] Please review and let me know if to make changes.

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @denigit5 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that hasn't any commit from your partner so kindly tell your partner to commit because this is a pair-programming activity

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by denigit5** [comment]
Hello. I have tried to reach out to my partner at our agreed time today but
they were not responding. I have tried to reach out to student success but
no reply. What should I do?

**Review by t-yanick — COMMENTED** [review]
# STATUS: INALID SUBMISSION

Hi @denigit5 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that hasn't any commit from your partner so kindly tell your partner to commit because this is a pair-programming activity

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by denigit5** [comment]
Screenshot (176)
Hello, as you review this code I want to let you know that I was the only collaborator due to some inconviniences.

File: script.js [hunk 1]
```diff
@@ -19,6 +19,30 @@ document.addEventListener('DOMContentLoaded', () => {
     document.querySelector('.navbar-links').classList.remove('navbar-links-active');
     document.querySelector('.navbar-links').classList.add('navbar-links-inactive');
   });
+  const name = document.getElementById('name');
+  const email = document.getElementById('email');
+  const text = document.getElementById('text');
+  const formfield = document.getElementById('form');
+  const formData = {
```
Comment (Lucas-Masaba) [inline script.js:26]: - [ ] When I enter my name, email and text and refresh the page, the data I entered is not stored in local storage and all the data I entered does not appear on the form. Please fix this :+1:

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @denigit5  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Well-written commit messages :writing_hand: 
- Descriptive pull request :scroll: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @denigit5  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Well-written commit messages :writing_hand: 
- Descriptive pull request :scroll: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**